smarts/core/vehicle_index.py

```diff
@@ -343,6 +284,37 @@ def relinquish_agent_control(self, sim, vehicle_id, social_vehicle_id):
 
         return vehicle
 
+    def recreate_agent_vehicle(
```

Comment:
Why is this a public method?

Reply:
I reverted it back to the original private method.